### DISCUSSION TITLE
Get controlplane config after interactive setup?

### DISCUSSION THREAD

**Discussion by jetersen** [siderolabs/talos#5282] [Q&A] [answered]
Is it possible to retrieve the config after interactive setup?

Or should I simply reinstall with non interactive?

**Comment by smira** [comment, marked as answer]
yes, it's possible:

1. `talosctl get mc -o yaml` and extract `.spec` field using `yq`
2. `talosctl read /system/state/config.yaml`

**Reply by jetersen** [reply]
Thanks!
